docs/content/operator/validator-node/configs/pruning.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Pruning
+description: How to prune your IOTA Full Node.
```

Comment:
```suggestion
description: How to prune your IOTA validator Node.
```

Reply:
Fixed in ede15b84a110e29ed56b96fd6ceade6336c84587